node/src/BaseClient.ts

```diff
@@ -3209,24 +3213,26 @@ export class BaseClient {
             hash: script.getHash(),
             keys: option?.keys?.map((item) => {
```

Comment:
I think conversion for keys and args is not needed at all

Reply:
Protobuf requires it, why isn't required in your opinion?

Reply:
Oh right. So you can simplify this
```suggestion
            keys: option?.keys?.map(k => Buffer.from(k));
```
If a `key` already a `Buffer` - nothing happens